src/components/Button/index.examples.js

```diff
@@ -8,14 +8,14 @@ const standardButtonExamples = [
   {type: 'primary', children: 'Primary'},
   {type: 'success', children: 'Success'},
   {type: 'warning', children: 'Warning'},
-  {type: 'danger', children: 'Danger'}
+  {type: 'danger', children: 'Danger'},
+  {type: 'orange', children: 'Orange'},
+  {type: 'turquoise', children: 'turquoise'}
```

Comment:
My only problem with `<Button small red pill outline>` is why are you using all those options? Do they have meaning for the user or is it just because you like red and the outline and pill styles better personally? `<DangerButton small>Danger</DangerButton>` makes more sense to me and implements  `red pill outline` behind the scenes.